### PR TITLE
fix(es): update 2 celebration names

### PR DESCRIPTION
**Pull request by miguelms95**
- our_lord_jesus_christ_king_of_the_universe
- our_lord_jesus_christ_the_eternal_high_priest

### CODE REVIEW

**Comment by TobiTenno** [comment]
@miguelms95 since everything from the first few commits was not changing anything (given the essential reverts), i dropped them off. lemme know if it seems like something's missing now, cause your old stuff already matched dev... i'm thinking they were old commits.

**Comment by TobiTenno** [comment]
@emagnier @tukusejssirs your reviews would be appreciated!

File: lib/locales/es.ts [hunk 1]
```diff
@@ -501,8 +501,8 @@ export const locale: Locale = {
     our_lady_of_the_valley: 'Nuestra Señora del Valle',
     our_lady_queen_of_peace: 'Nuestra Señora, Reina de la Paz',
     our_lady_refuge_of_sinners: 'Nuestra Señora, refugio de los pecadores',
-    our_lord_jesus_christ_king_of_the_universe: 'Cristo Rey',
-    our_lord_jesus_christ_the_eternal_high_priest: 'Nuestro Señor Jesucristo, Sumo y Eterno presbítero',
+    our_lord_jesus_christ_king_of_the_universe: 'Cristo Rey del Universo', // src: https://textosparalaliturgia.blogspot.com/2014/09/misal-romano-jesucristo-rey-del-universo.html
```
Comment (tukusejssirs) [inline lib/locales/es.ts]: The source says the celebration name is `Jesucristo, Rey del Universo`.

```suggestion
    our_lord_jesus_christ_king_of_the_universe: 'Jesucristo, Rey del Universo', // src: https://textosparalaliturgia.blogspot.com/2014/09/misal-romano-jesucristo-rey-del-universo.html
```

File: lib/locales/es.ts [hunk 2]
```diff
@@ -501,8 +501,8 @@ export const locale: Locale = {
...
+    our_lord_jesus_christ_the_eternal_high_priest: 'Nuestro Señor Jesucristo, Sumo y Eterno sacerdote', // src: https://textosparalaliturgia.blogspot.com/2014/05/misal-romano-jesucristo-sumo-y-eterno.html
```
Comment (tukusejssirs) [inline lib/locales/es.ts]: The source says the celebration name is `Jesucristo, Sumo y Eterno Sacerdote`.

```suggestion
    our_lord_jesus_christ_the_eternal_high_priest: 'Jesucristo, Sumo y Eterno Sacerdote', // src: https://textosparalaliturgia.blogspot.com/2014/05/misal-romano-jesucristo-sumo-y-eterno.html
```

**Review by tukusejssirs — CHANGES_REQUESTED** [review]
The celebration names are not cited carefully. Other then that, LGTM.

Thanks, @miguelms95 and @TobiTenno! :pray:

**Comment by TobiTenno** [comment]
please stop making the title very long. this causes issues like wrapping in the release message itself, and is a problem as it stands right now with our config. the config update has not been merged, but i still don't think we should behaviorally remove the limit on the title length, it invites a difficult and chaotic trend, making things very difficult to follow

**Review by tukusejssirs — CHANGES_REQUESTED** [review]
LGTM, thanks! :pray:

**Review by tukusejssirs — APPROVED** [review]

**Pull request merged**